rpcs3/Emu/Cell/lv2/sys_sm.cpp

```diff
@@ -41,9 +43,40 @@ error_code sys_sm_get_ext_event2(vm::ptr<u64> a1, vm::ptr<u64> a2, vm::ptr<u64>
 	return not_an_error(CELL_EAGAIN);
 }
 
-error_code sys_sm_shutdown(u16 op, vm::ptr<void> param, u64 size)
+error_code sys_sm_shutdown(ppu_thread& ppu, u16 op, vm::ptr<void> param, u64 size)
 {
-	sys_sm.todo("sys_sm_shutdown(op=0x%x, param=*0x%x, size=0x%x)", op, param, size);
+	sys_sm.trace("sys_sm_shutdown(op=0x%x, param=*0x%x, size=0x%x)", op, param, size);
```

Comment:
Please avoid using trace in syscalls, it is a workaround for massive log spam. In a perfect world we wouldn't need it. Here it is success.

Reply:
Will change it.
By the way, should I add `ppu.state += cpu_flag::wait` at the beginning of this syscall?

Reply:
Yes.

Reply:
> Please avoid using trace in syscalls, it is a workaround for massive log spam. In a perfect world we wouldn't need it. Here it is success.

We have already had at least one .success() log in switch...case, do we still need to make the call log ("sys_sm_shutdown(op=0x%x, param=*0x%x, size=0x%x)") `success`?

Reply:
Doesn't hurt, and it has unused params so better to them.

Reply:
Done.